mri/utils/process_SWI

```diff
@@ -0,0 +1,123 @@
+# -*- coding: utf-8 -*-
+##########################################################################
+# pySAP - Copyright (C) CEA, 2017 - 2018
+# Distributed under the terms of the CeCILL-B license, as published by
+# the CEA-CNRS-INRIA. Refer to the LICENSE file or to
+# http://www.cecill.info/licences/Licence_CeCILL-B_V1-en.html
+# for details.
+##########################################################################
+
+import numpy as np
+
+
+def virtual_coil_reconstruction(imgs):
```

Comment:
The term "reconstruction" is disturbing in the pysap-mri context, when in fact this is a coil "combination" method.

Reply:
> I don't actually intend to add SWI-related code in #115. I feel that current PR should still be good to merge.

Ahh, than we really need someone to work on these codes, maybe since you use them, can you take over from here? I would just want to ensure that we have this at main branch. We can make changes to this branch or maybe get a new branch for this.